### ISSUE TITLE
Add flag to not create subprocess groups

### ISSUE THREAD

**Issue by Aleksej-Kozlov** [watchexec/watchexec#158]
Example one-line CMD script:
`watchexec.exe --postpone start "" /min cmd /k echo %%WATCHEXEC_WRITTEN_PATH%%`
After first update I see background window, as expected. After second update (without closing that window) I see nothing. Expected that second background window should appear. Now second window appears only after closing the first.

**Comment by passcod** [comment]
Nope, what you're seeing is expected behaviour, we don't detach child processes. You can use `--restart` to have watchexec kill the process on change, or you can run your own command detached using `at` or something.

**Comment by Aleksej-Kozlov** [comment]
Please note that I used the "start" command in the script. This command detaches child process if called without the "/wait" parameter.

**Comment by passcod** [comment]
Can you run with verbose output and show that?

Also, what happens if you use `--restart`?

**Comment by Aleksej-Kozlov** [comment]
Here is more complicated example.
In command line: `watchexec.exe --postpone 2.cmd`
where 2.cmd is:
```@echo off
echo %DATE% %TIME% before
start "" pause
echo %DATE% %TIME% after
```
Then at 11:00 in command line I have executed 2 times `touch temp`.
One window with `pause` appeared. At 11:03 I have closed this window and another window with `pause` immediately appeared. Log in console:
```
24.06.2020 11:00:37,94 before
24.06.2020 11:00:37,94 after
24.06.2020 11:03:22,39 before
24.06.2020 11:03:22,39 after
```
Please pay attention to the second line. Script 2.cmd closed at 11:00 and control returned to watchexec.exe. However watchexec.exe is awaiting something else. Perhaps it waiting for the completion of the entire tree of running subprocesses.

**Comment by passcod** [comment]
The case I'm trying to disambiguate is whether control is indeed returned to watchexec or whether the subprocess or group has the hand. A verbose run may have enough information to see this. My understanding is that we _do_ have things around subprocess groups for the purpose of correctly killing the entire subprocess to _avoid_ having two instances running.

**Comment by Aleksej-Kozlov** [comment]
Here is log with `--verbose`.
```
[...]
*** built glob set; 0 literals, 0 basenames, 0 extensions, 0 prefixes, 0 suffixes, 0 required extensions, 9 regexes
*** Watching "\\\\?\\D:\\git\\tacho__portal__portal-ui 1"
*** Waiting for filesystem activity
*** Paths updated: [PathOp { path: "\\\\?\\D:\\git\\tacho__portal__portal-ui 1\\temp", op: Some(WRITE), cookie: None }]
*** Launching child process
*** Assembled command "cmd.exe" "/C" "2.cmd"
*** Waiting for filesystem activity
24.06.2020 11:25:09,22 before
24.06.2020 11:25:09,23 after
*** Paths updated: [PathOp { path: "\\\\?\\D:\\git\\tacho__portal__portal-ui 1\\temp", op: Some(WRITE), cookie: None }]
*** Waiting for process to exit...
[here it's awaiting until I close the 'pause' window]
*** Launching child process
*** Assembled command "cmd.exe" "/C" "2.cmd"
*** Waiting for filesystem activity
24.06.2020 11:27:37,70 before
24.06.2020 11:27:37,70 after
```
I.e. the subprocess with "2.cmd" closed, but watchexec also awaits until the sub-subprocess started from "2.cmd" is closed. I believe watchexec should await only subprocess which it directly created, not all sub-subprocesses. Or, maybe, it should be regulated by command-line switch.

**Comment by passcod** [comment]
>  believe watchexec should await only subprocess which it directly created, not all sub-subprocesses.

As I said, no, this is specifically something watchexec does and should do, because "keeping track of all subprocesses" is the most commonly wanted behaviour. Changing that default would be a major change and everyone _else_ would complain.

I can get behind adding a switch to disable this, though.

**Comment by passcod** [comment]
Relabel: the feature request for a switch isn't windows-specific

**Comment by passcod** [comment]
Released in 1.17.0
